docs/Configuration.md

```diff
@@ -146,7 +176,41 @@ to specifying a profile JSON file containing the entire
 
 ## ZONEMASTER section
 
-TBD
+The ZONEMASTER section has several keys :
+`max_zonemaster_execution_time`,
+`number_of_professes_for_frontend_testing`,
+`number_of_professes_for_batch_testing`,
+`force_hash_id_use_in_API_starting_from_id`, `lock_on_queue`,
+`maximal_number_of_retries`.
+
+### max_zonemaster_execution_time
+
+This key allows to define the interval in seconds to reload unfinished
+queries.
+This assume that after `max_zonemaster_execution_time`, if the test is
+not finished, it has fail. And it will be run again up to
+`maximal_number_of_retries`.
+
+### maximal_number_of_retries
+
+Specifies the number of time a test is allowed to be run again if
+unfinished after `max_zonemaster_execution_time`.
+
+### number_of_professes_for_frontend_testing
+
+not used in the code
+
+### number_of_professes_for_batch_testing
+
+not used in the code
+
+### force_hash_id_use_in_API_starting_from_id
+
+used -> todo
```

Comment:
@mtoma, can you give a description and legal values for this?

Reply:
There is a comment about this property in a [Travis .ini file](https://github.com/zonemaster/zonemaster-backend/blob/159e80bc3510582f0d3f028a60a2d44773f19419/share/travis_postgresql_backend_config.ini#L24..L29)

So it looks like a condition to have a database with `id` made of integers and hashes. The value of this key is the boundary between each range.